lxd/storage/drivers/generic_vfs.go

```diff
@@ -170,7 +170,14 @@ func genericVFSMigrateVolume(d Driver, s *state.State, vol Volume, conn io.ReadW
 		path := shared.AddSlash(mountPath)
 
 		d.Logger().Debug("Sending filesystem volume", log.Ctx{"volName": vol.name, "path": path, "bwlimit": bwlimit, "rsyncArgs": rsyncArgs})
-		return rsync.Send(vol.name, path, conn, wrapper, volSrcArgs.MigrationType.Features, bwlimit, s.OS.ExecPath, rsyncArgs...)
+		err := rsync.Send(vol.name, path, conn, wrapper, volSrcArgs.MigrationType.Features, bwlimit, s.OS.ExecPath, rsyncArgs...)
+
+		status, _ := shared.ExitStatus(err)
```

Comment:
I wasn't sure what to do about the returned error here. If the exit status can't be found then `status` will be `-1` and this would revert to the original behaviour. However, I could also ensure that the error is nil in the following if statement if required.

Reply:
I think the current logic is fine here, if we can find the exit status and it's 24 + we have the flag set, then we ignore the error, otherwise we process the error as we do today.